xml/Microsoft.Extensions.Logging/ILogger.xml

```diff
@@ -141,7 +141,7 @@
         <param name="eventId">Id of the event.</param>
         <param name="state">The entry to be written. Can also be an object.</param>
         <param name="exception">The exception related to this entry.</param>
-        <param name="formatter">Function to create a <see cref="T:System.String" /> message of the <paramref name="state" /> and <paramref name="exception" />.</param>
+        <param name="formatter">Function to create a <see cref="T:System.String" /> message of the <paramref name="state" /> and <paramref name="exception" />. **Note: When using extension methods such as <see cref="M:Microsoft.Extensions.Logging.LoggerExtensions.Log(Microsoft.Extensions.Logging.ILogger,Microsoft.Extensions.Logging.LogLevel,Microsoft.Extensions.Logging.EventId,System.Exception,System.String,System.Object[])"/> and <see cref="M:Microsoft.Extensions.Logging.LoggerExtensions.LogError(Microsoft.Extensions.Logging.ILogger,System.Exception,System.String,System.Object[])"/>, the function passed here will not add the <paramref name="exception"/> parameter to the log message. When using these overloads, implementations should explicitly append the <paramref name="exception"/> to the log message.**</param>
```

Comment:
>. [](http://example.com/codeflow?start=158&length=1)

would saying something like the following will be simpler and clearer? 

```
The default formatter in the logger extensions doesn't include exceptions when formatting messages. It's essential for the ILogger.Log provider implementation to decide how exceptions should be incorporated into the logs.
```

Reply:
By the way, thanks a lot for helping improving the docs.

Reply:
No problem! Will modify based on your suggestion

Reply:
Let me know if it looks good now